dotcom-rendering/src/web/components/FixedSmallSlowI.tsx

```diff
@@ -0,0 +1,29 @@
+import type { DCRContainerPalette } from '../../types/front';
+import { FrontCard } from './FrontCard';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedSmallSlowI = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	const trail = trails[0];
+
+	return (
+		<FrontCard
```

Comment:
Visually it makes no difference wrapping all this in a <UL> and <LI>. I was tempted to keep it all wrapped in a <UL> <LI> for consistency but decided against as we already have some precedent for not using UL LI in Thrashers